easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-2013.5.1-ictce-5.4.0.eb

```diff
@@ -0,0 +1,20 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+##
```

Comment:
No, if there is a custom easyblock, it will be picked up.

Jenkins is just complaining that it doesn't know this block.